Framework/PythonInterface/mantid/_plugins/src/curvefitting.cpp.in

```diff
@@ -0,0 +1,24 @@
+/*****************************************************************************************/
+/********** PLEASE NOTE! THIS FILE WAS AUTO-GENERATED FROM CMAKE.  ***********************/
+/********** Source = curvefitting.cpp.in **********************************************************/
+/*****************************************************************************************/
+#include <boost/python/def.hpp>
+#include <boost/python/module.hpp>
+#include <boost/python/docstring_options.hpp>
+
+// See http://docs.scipy.org/doc/numpy/reference/c-api.array.html#PY_ARRAY_UNIQUE_SYMBOL
```

Comment:
Remove these 3 lines (starting with last line shown) causes a build failure concerning ``_import_array`` in the generated file.